dev/droidmate/projects/core/src/main/groovy/org/droidmate/android_sdk/AdbWrapper.groovy

```diff
@@ -640,9 +655,30 @@ public class AdbWrapper implements IAdbWrapper
     try
     {
 
-      sysCmdExecutor.execute(commandDescription, cfg.adbCommand,
-        "-s", deviceSerialNumber,
-        "pull", pulledFilePath, destinationFilePath)
+      if (runAsPackage == null)
+        sysCmdExecutor.execute(commandDescription, cfg.adbCommand,
+          "-s", deviceSerialNumber,
+          "pull", pulledFilePath, destinationFilePath)
+      else
+      {
+        // To solve a problem with permissions (enhanced security on Android 6)
```

Comment:
Ah, here is the explanation I needed above. Put it in javadoc of one method, link the javadoc of other method to the first one.